packages/@coorpacademy-progression-engine/src/compute-next-step/compute-next-step.js

```diff
@@ -431,8 +431,20 @@ export const computeNextStepForReview = (
         state.pendingSlides
       );
 
-      // all other questions have been already right answered, so we close the progression
+      // all other questions have been already right answered BUT you fail the last question
+      if (!pendingSlide && !isCorrect) {
```

Comment:
could you add a test that takes `!pendingSlide && !isCorrect` scenario into consideration?